agrirouter-sdk-dotnet-standard-api/Dto/Onboard/VerificationRequest.cs

```diff
@@ -0,0 +1,52 @@
+using Newtonsoft.Json;
+
+namespace Agrirouter.Api.Dto.Onboard
+{
+    /// <summary>
+    ///     Data transfer object for the communication.
+    /// </summary>
+    public class VerificationRequest
+    {
+        /// <summary>
+        ///     External ID.
+        /// </summary>
+        [JsonProperty(PropertyName = "id")]
+        public string ExternalId { get; set; }
```

Comment:
see above, why diverge here?

Reply:
This makes more sense, since the term "external id" is way more convenient as the term "id". Would not change that. Please resolve if you agree.